src/goto-instrument/cover_basic_blocks.cpp

```diff
@@ -155,6 +149,18 @@ void cover_basic_blockst::output(std::ostream &out) const
         << '\n';
 }
 
+void cover_basic_blockst::add_block_lines(
+  cover_basic_blockst::block_infot &block,
```

Comment:
:pick: should imho be called `block_info`.

Reply:
IMHO, it would be better to rename `block_infot` to `blockt`. This is on the basis that you wouldn't bother creating instances of a struct that didn't contain information. Therefore the word `info` is redundant in the context of the name. It doesn't really convey any useful meaning to the reader.